packages/slate-editor/src/modules/editor-v4/EditorV4.tsx

```diff
@@ -447,8 +464,27 @@ const EditorV4: FunctionComponent<EditorV4Props> = (props) => {
                         containerRef={containerRef}
                         onClose={closeFloatingStoryEmbedInput}
                         onRootClose={rootCloseFloatingStoryEmbedInput}
-                        onSubmit={submitFloatingStoryEmbedInput}
-                        renderInput={withStoryEmbeds.renderInput}
+                        renderInput={() =>
+                            withStoryEmbeds.renderInput({
+                                onCreate: submitFloatingStoryEmbedInput,
+                                onRemove: closeFloatingStoryEmbedInput,
```

Comment:
> `onCreate` and `onRemove`

I wonder if these names precisely describe what the properties do. How about `onSubmit` and `onClose` ?